docs/manual/working/scalaGuide/main/sql/ScalaAnorm.md

```diff
@@ -442,6 +442,14 @@ SQL("UPDATE item SET last_modified = {mod} WHERE id = {id}").on(params:_*)
 It's not recommanded because moreover hiding implicit resolution issues, as untyped it could lead to runtime conversion error, with values are passed on statement using `setObject`.
 In previous example, `java.util.Date` is accepted as parameter but would with most databases raise error (as it's not valid JDBC type).
 
+In some cases, some JDBC drivers returns a result set positioned on the first row rather than [before this first row](http://docs.oracle.com/javase/7/docs/api/java/sql/ResultSet.html) (e.g. stored procedured with Oracle JDBC driver).
```

Comment:
Doc about `.withResultSetOnFirstRow`, used to force Anorm to consider non-standard ResultSet is initially positioned on first row (not before).